.changeset/chatty-lizards-reflect.md

```diff
@@ -0,0 +1,17 @@
+---
+"@rocket.chat/meteor": major
+"@rocket.chat/ddp-client": major
+---
+
+Removes deprecated livechat:transfer methods
+
+Removes the deprecated livechat:closeRoom method
```

Comment:
```suggestion
```

There's a changeset for `livechat:closeRoom` already, safe to remove it from this one.